src/sage/matroids/database_matroids.py

```diff
@@ -2116,7 +2116,8 @@ def Spike(r, t=True, C3=[]):
     EXAMPLES::
 
         sage: M = matroids.Spike(3, False); M
-        Free 3-spike\t: Matroid of rank 3 on 6 elements with 20 bases
+        Free 3-spike\t: M \ {'t'}, where M is Matroid of rank 3 on 7 elements
+        with 3 non-spanning circuits
```

Comment:
```suggestion
         with 3 non-spanning circuits
```
Just as a subtle nod to the fact it is a continuing doctest (this sort of thing is common in Sage's doctests; IMO, it should be more universal).

Reply:
I went ahead and did many docstring changes. I changed some example blocks having in mind a 95th column limit, and I made the REFERENCE blocks into one-liners where applicable.